src/renderer/components/Wallet/WalletDetails/WalletDetails.vue

```diff
@@ -182,10 +182,10 @@ export default {
     },
 
     isOwned () {
-      const wallet = this.$store.getters['wallet/byAddress'](this.currentWallet.address)
-      const wallets = this.$store.getters['wallet/byProfileId'](this.session_profile.id)
-
-      return wallets.includes(wallet)
+      return [
+        ...this.$store.getters['wallet/byProfileId'](this.session_profile.id),
+        ...this.$store.getters['ledger/wallets']
+      ].some(wallet => wallet.address === this.currentWallet.address)
```

Comment:
`find` returns the element, while `some` return a bool, which is all that's needed.

Reply:
I know the difference, but this is more about being verbose to the developer, without having to remember what "some" does. "find" is self-explanatory imo 🤷‍♂️

Reply:
Sorry, I misunderstood the question. Imho `find` isn't that clear either, and I could imagine someone thinking it will return a bool instead of the value (in the sense of: `yes - the element was found`). In the worst case the answer is a google search away.

Reply:
yeh this was more food for thought than to actually change. personally i don't think `some` is clear at all in what it does, but we can go with it. I'll test a bit later on @j-a-m-l once I have my Ledger to hand 👌

Reply:
Initially I was using `find` and then, on a new line, compare the found wallet and `this.currentWallet`, but I realized that using `some` was simpler.